modules/vba_documents/app/controllers/vba_documents/docs/v0/api_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module VBADocuments
+  module Docs
+    module V0
+      class ApiController < ApplicationController
+        include Swagger::Blocks
```

Comment:
Let's kill the swagger, we should use the raw yaml for the v3 spec; not sure how to include it in the implementation yet.